test/els_test_utils.erl

```diff
@@ -71,23 +76,27 @@ init_per_suite(Config) ->
   DiagnosticsIncludePath = filename:join([ RootPath
                                          , <<"include">>
                                          , <<"diagnostics.hrl">>]),
+  DiagnosticsBehaviourPath = filename:join([ RootBehaviourPath
```

Comment:
Not related to this PR, so feel free to ignore it, but we should probably refactor this module and get rid of the duplications in this module.